lib/landrush/cap/host/windows/configure_visibility_on_host.rb

```diff
@@ -1,5 +1,6 @@
 # This file needs to be able to execute out of the Vagrant context. Do not require any non core or non relative files
 require 'ipaddr'
+require 'English'
```

Comment:
Why is that needed?

Reply:
Because of this:

    lib/landrush/cap/host/windows/configure_visibility_on_host.rb:29:13: C: Style/SpecialGlobalVars: Prefer $CHILD_STATUS from the stdlib 'English' module (don't forget to require it) over $?.
            $?.exitstatus.zero?
            ^^

Reply:
ahh, got you.

Reply:
ok